README.md

```diff
@@ -15,12 +15,12 @@ from the ~/.aws/config file.
 
 See [Setting Region](https://docs.aws.amazon.com/sdk-for-javascript/v2/developer-guide/setting-region.html) page for more information.
 
-### TypeScript 3.5.x
+### TypeScript 3.8.x
```

Comment:
Honestly we don't even need 3.5. The real version we need is some undetermined previous version.

Figured we'd align up our documentation and all and 3.8 was what I was playing with testing hash private functions. This ultimately doesn't matter because it's an independent dev dependency.